bounded-core/src/main/scala/io/cafienne/bounded/aggregate/AggregateRootActor.scala

```diff
@@ -48,14 +48,14 @@ trait AggregateRootActor extends PersistentActor with AggregateRootStateCreator
     * @param command is the commands your aggregate root expects
     * @param currentState of the Aggregate Root
     * @return a seqency of events when everything is Right, or an Exception (Left)
-    */
+    */                                                                                //TODO replace Either (no EXCEPTION)
   def handleCommand(command: AggregateRootCommand, currentState: AggregateRootState): Either[Exception, Seq[AggregateRootEvent]]
 
   /**
     * In your implementation you can handle your own commands by writing a commandHanlder {} function.
     * @param next is the partial Actor.Receive function of messages you like to handle next to the Aggregate Root default messages
     */
-  final def commandHandler(next: Actor.Receive): Unit = { commandReceivers = commandReceivers orElse next }
```

Comment:
Hmm, not sure what this means.. commented code can be thrown away, or accidentally commented?

Reply:
part of an extension pattern that allowed to create your own command handlers. It's not needed and I've removed it completely, including the supporting structure underneath.